tools/unirestore/README.md

```diff
@@ -0,0 +1,99 @@
+# Universal Restore Utility
+
+Restore a backup of a set of SSTable files taken from any given Scylla/Cassandra cluster to any size/topology cluster. 
+
+
+Overview of what it does and how it does it:
+
+As a user, I have a set of sstable files I have backed up from my old cluster. The set looks like this[1]:
+
+`root-dir/host/keyspace/table/sstable_files`
+
+Also, the user provides me with:
+
+- the original schema dump in a CQL file (can be sanitized to include only keyspaces/tables)
+- ssh credentials for a sudo user on all nodes (key and password access are supported)
+- One of the node IPs in the cluster for cluster metadata gathering
+- If needed, API URL and CQLSH credentials
+
+I have a new cluster into which I'd like to load the data. The node count and topology is irrelevant (though if the new cluster 
+is multi-DC based, it would make sense to limit the destination node list to a local DC and adjust the replicas after restore, 
+in order to avoid cross DC traffic)
+
+I assume the schema has already been loaded into the new cluster.
+
+## The workflow:
```

Comment:
This is very helpful, both to reviewers and to users who wish to track progress.

Reply:
/cc @mmatczuk